draft-ietf-tls-semistatic-dh.md

```diff
@@ -236,45 +230,29 @@ Finished using the Master Secret. These MACs serve different
 purposes: the first authenticates the handshake and the second proves
 possession of the ephemeral secret.
 
-
-## Key Schedule
-
-The final HKDF-Extract stage of the TLS 1.3 key schedule has
-an HKDF-Extract with the IKM of 0. When static key exchange
-is negotiated, that 0 is replaced with SS, as shown below.
-
-~~~~
-...
-           Derive-Secret(., "derived", "")
-                 |
-                 v
-     SS -> HKDF-Extract = Master Secret
-                 |
-                 +-----> Derive-Secret(., "c ap traffic",
-                 |                     ClientHello...server Finished)
-                 |                     = client_application_traffic_secret_0
-                 |
-...
-~~~~
-
 # Client Authentication
 
-[[OPEN ISSUE]] In principle, we can do client authentication the same way,
-with the client's DH key in Certificate and a MAC in CertificateVerity.
-However, it's less good because the client's static key doesn't get mixed
-in at all. Also, client DH keys seem even further off.
+Client authentication works similar to that of server authentication described
+in {{protocol}}. In particular, servers indicate support of semi-static keys
+by sending one of the relevant SignatureScheme values defined in {{negotiation}}
+inside the CertificateRequest "signature_algorithms" extension. If applicable,
+clients reply with a non-empty Certificate message carrying a corresponding certificate
+with static DH key matching the chosen signature algorithm. Clients then also
+change the computation of their CertificateVerify message to match described in
+{{cert-verify}}, with the exception that the transcript hash Handshake Context
+matches that described in {{!RFC8446}}, Section 4.4.
 
-# Security Considerations
+If no matching certificate is available, clients send an empty Certificate message as
+per {{!RFC8446}}; Section 4.4.2.
 
-[[OPEN ISSUE: This design requires formal analysis.]]
```

Comment:
Don't remove this. We still need formal analysis! So leave this in or have a cite to that analysis.